environment.yml

```diff
@@ -4,8 +4,8 @@ channels:
   - conda-forge
   - cta-observatory
 dependencies:
-  - ctapipe=0.7.0
-  - gammapy=0.8
+  - ctapipe
```

Comment:
Please, specify 0.9.1 here.

I plan to support directly >= 0.10.1 in the release after the one that will contain the changes from this PR and PR #83